### ISSUE TITLE
v1.4.0 no longer autopopulates 

### ISSUE THREAD

**Issue by gsophy** [hashicorp/vscode-terraform#232]
Upgraded to version 1.4.0 of the Terraform extension and the auto-complete features no longer work.  When typing in a property type (resource, for example), the resource name will complete, but no additional details fill in (resource type, name or brackets).  

Rolling back to version 1.3.12 restored the functionality.  

Specs:
MacOS Catalina
VSCode 1.39
tflint path :  /usr/local/bin/tflint
terraform path /usr/local/bin/terraform
Terraform version: v0.12.9

**Comment by sdcscripts** [comment]
This has stopped working for me too, no autocomplete of resource type, name and brackets :(

**Comment by bcastilho90** [comment]
It doesn't work for me on 1.4.0 either.
WSL 2
VS Code - 1.39.2 Remote Extension
Terraform v0.12.13

**Comment by mauve** [comment]
terraform 0.12 requires using the language server

**Comment by opsline-ilan** [comment]
^ I have the LS installed, running, everything updated, and still no autocomplete.

**Comment by beerygaz** [comment]
Same issue, the output in VS code for the language server just returns:

2019/12/12 07:32:09 Server started
2019/12/12 07:32:14 (string) (len=23) "With Error or No Config"
2019/12/12 07:32:14 (hcl.Diagnostics) no diagnostics

**Comment by stuzlogle** [comment]
As mentioned in #253. Try setting terraform.languageServer.pathToBinary

e.g.

```
    "terraform.languageServer": {
        "enabled": true,
        "args": [],
        "pathToBinary": "/<$HOME>/.vscode/extensions/mauve.terraform-1.4.0/lspbin"
    },
```

Where $HOME is your home directory.

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
